Underhanded contest 2022

chriseth opened the pull request.

franzihei committed:
update to 2022 framework

franzihei committed:
further updates

chriseth committed:
Provide theme.

hrkrshnn commented on .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910:
```diff
@@ -0,0 +1 @@
+I"O{"source"=>"/Users/franzihei/Documents/github/solidity-underhanded-contest", "destination"=>"/Users/franzihei/Documents/github/solidity-underhanded-contest/_site", "collections_dir"=>"", "cache_dir"=>".jekyll-cache", "plugins_dir"=>"_plugins", "layouts_dir"=>"_layouts", "data_dir"=>"_data", "includes_dir"=>"_includes", "collections"=>{"posts"=>{"output"=>true, "permalink"=>"/:categories/:y ...
```
I think the cache can be removed. Maybe put it in a gitignore? https://www.toptal.com/developers/gitignore/api/jekyll

franzihei replied:
yes I will try to do that!

franzihei committed:
update to 2022

- adding 2020 website as ``/2020``
- adding judges (to be confirmed)
- adding timeline
- adjusting some wording
- adding gitignore for jekyll stuff

franzihei committed:
small fixes

franzihei committed:
updating prizes section

franzihei committed:
more adjustments

franzihei committed:
resolving conflicts

franzihei committed:
Merge branch 'master' into underhanded-contest-2022

franzihei committed:
update judges

franzihei committed:
button update

franzihei commented:
Sorry for the mess I made here, lol! I think it should be all cleaned up now. :)

franzihei committed:
Update index.html

franzihei committed:
fixing mixups

franzihei committed:
adding board of fame and proofreading everything

franzihei committed:
further optimizations

franzihei committed:
fixes and adding links to winning submissions

franzihei committed:
adding time to deadline
